test/gateways/authorize_net_test.exs

```diff
@@ -29,27 +29,43 @@ defmodule Gringotts.Gateways.AuthorizeNetTest do
     config: %{name: "64jKa6NA", transactionKey: "4vmE338dQmAN6m7B"},
     refId: "123456",
     order: %{invoiceNumber: "INV-12345", description: "Product Description"}, 
-    lineItem: %{itemId: "1", name: "vase", description: "Cannes logo", quantity: "18", unitPrice: "45.00" }
+    lineItem: %{itemId: "1", name: "vase", description: "Cannes logo", quantity: "18", unitPrice: "45.00"}
+  ]
+  @opts_refund [
+    config: %{name: "64jKa6NA", transaction_key: "4vmE338dQmAN6m7B"}, 
+    ref_id: "123456", 
+    payment: %{card: %{number: "5424000000000015", year: 2020, month: 12}}
+  ]
+  @opts_store [
+    config: %{name: "64jKa6NA", transaction_key: "4vmE338dQmAN6m7B"}, 
+    profile: %{merchant_customer_id: "123456", description: "Profile description here", email: "customer-profile-email@here.com"}
+  ]
+  @opts_store_no_profile [
+    config: %{name: "64jKa6NA", transaction_key: "4vmE338dQmAN6m7B"}, 
   ]
 
+  @refund_id "60036752756"
+  @void_id "60036855217"
+  @unstore_id "1813991490"
+
   describe "purchase" do
-    test "test_successful_purchase" do
-      with_mock HTTPoison, 
+    test "successful response" do
```

Comment:
it should become something like `test purchase successful response with all correct params` or `test purchase with all correct params` or some variant of that, should convey a meaning.